test/integration/agent/flowexporter_test.go

```diff
@@ -127,8 +127,13 @@ func TestConnectionStoreAndFlowRecords(t *testing.T) {
 	ifStoreMock := interfacestoretest.NewMockInterfaceStore(ctrl)
 	// TODO: Enhance the integration test by testing service.
 	connStore := connections.NewConnectionStore(connDumperMock, ifStoreMock, nil, nil, testPollInterval)
+	// Hard-coded conntrack occupancy metrics for test
+	testConnectionMetric := &flowexporter.ConntrackOccupancy{
+		MaxConnections:   300000,
+		TotalConnections: 2,
+	}
 	// Expect calls for connStore.poll and other callees
-	connDumperMock.EXPECT().DumpFlows(uint16(openflow.CtZone)).Return(testConns, nil)
+	connDumperMock.EXPECT().DumpFlows(uint16(openflow.CtZone)).Return(testConns, testConnectionMetric, nil)
```

Comment:
can't you pass nil here? testConnectionMetric is not used anywhere right? It is better to avoid as there is no metric test here.

Reply:
Thanks, done.